gtsfm/utils/geometry_comparisons.py

```diff
@@ -205,3 +205,50 @@ def compute_relative_unit_translation_angle(U_1: Optional[Unit3], U_2: Optional[
     angle_rad = np.arccos(dot_product)
     angle_deg = np.rad2deg(angle_rad)
     return angle_deg
+
+
+def compute_translation_to_direction_angle(
+    i2Ui1: Optional[Unit3],
+    wTi2: Optional[Point3],
+    wTi1: Optional[Pose3]
+) -> Optional[float]:
+    """Compute the angle between a translation direction and point.
```

Comment:
is this docstring accurate? no point is passed in, right? just 2 poses and a translation dir?

Reply:
Updated it, hope its better now.

Reply:
yes, thanks